src/Mailer/PostmarkMailer.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace ZfrMail\Mailer;
+
+use GuzzleHttp\ClientInterface;
+use ZfrMail\Mail\MailInterface;
+use ZfrMail\Mail\RenderedMailInterface;
+use ZfrMail\Mail\TemplatedMailInterface;
+
+/**
+ * @author Michaël Gallego
+ */
+class PostmarkMailer implements MailerInterface
+{
+    const POSTMARK_BASE_API = 'https://api.postmarkapp.com';
+
+    /**
+     * @var string
+     */
+    private $serverToken;
+
+    /**
+     * @var ClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @param string          $serverToken
+     * @param ClientInterface $httpClient
+     */
+    public function __construct(string $serverToken, ClientInterface $httpClient)
+    {
+        $this->serverToken = $serverToken;
+        $this->httpClient  = $httpClient;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function send(MailInterface $mail)
+    {
+        if ($mail instanceof RenderedMailInterface) {
+            $this->sendRenderedMail($mail);
+        } elseif ($mail instanceof TemplatedMailInterface) {
```

Comment:
You can avoid else keyword here with a `return;` statement (object calisthenics :P)